src/servlet/ServletPage.java

```diff
@@ -0,0 +1,20 @@
+package servlet;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletContext;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class ServletPage extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        String path = "/WEB-INF/page.html";
```

Comment:
Are you sure you have such path in your code base? Couldn't figure out where the `WEB-INF` folder at all )) 
![image](https://user-images.githubusercontent.com/48795460/76831839-261bc480-6839-11ea-869a-87e966385a57.png)
Or, am I looking the wrong way?)

Reply:
The assignment was prescribed that index.html should be in the WEB-INF folder.

Reply:
Yes, sure! I just could not find no WEB-INF folder in your branch! Did I miss it?

Reply:
I added it in new commit